tests/fixtures/vcr_cassettes/test_location_cli.yaml

```diff
@@ -0,0 +1,362 @@
+        srsName=\"http://www.opengis.net/gml/srs/epsg.xml#4326\" xmlns:gml=\"http://www.opengis.net/gml\">\n   <gml:outerBoundaryIs>\n      <gml:LinearRing>\n         <gml:coordinates>81.90748103627195,68.62188393530005
+        81.95644737476664,75.62478835890376 80.97284377382584,75.55719378094982 80.92925761838922,69.30716094300509
+        81.90748103627195,68.62188393530005</gml:coordinates>\n      </gml:LinearRing>\n   </gml:outerBoundaryIs>\n</gml:Polygon>"},{"name":"footprint","content":"MULTIPOLYGON
+        (((69.30716094300509 80.92925761838922, 75.55719378094982 80.97284377382584,
+        75.62478835890376 81.95644737476664, 68.62188393530005 81.90748103627195,
+        69.30716094300509 80.92925761838922)))"},{"name":"format","content":"SAFE"},{"name":"processingbaseline","content":"02.14"},{"name":"platformname","content":"Sentinel-2"},{"name":"filename","content":"S2B_MSIL2A_20200623T111629_N0214_R037_T43XDL_20200623T130334.SAFE"},{"name":"instrumentname","content":"Multi-Spectral
+        Instrument"},{"name":"instrumentshortname","content":"MSI"},{"name":"size","content":"762.70
+        MB"},{"name":"s2datatakeid","content":"GS2B_20200623T111629_017223_N02.14"},{"name":"producttype","content":"S2MSI2A"},{"name":"platformidentifier","content":"2017-013A"},{"name":"orbitdirection","content":"ASCENDING"},{"name":"platformserialidentifier","content":"Sentinel-2B"},{"name":"processinglevel","content":"Level-2A"},{"name":"identifier","content":"S2B_MSIL2A_20200623T111629_N0214_R037_T43XDL_20200623T130334"},{"name":"uuid","content":"cb3d6b7d-6938-4eb8-862e-1b592d1381a5"}]}]}}'
+    headers:
+      access-control-allow-credentials:
+      - 'true'
+      access-control-allow-origin:
+      - ''
+      content-length:
+      - '19276'
+      content-type:
+      - application/json
+      date:
+      - Tue, 23 Jun 2020 16:32:27 GMT
+      pragma:
+      - no-cache
+      server:
+      - Apache-Coyote/1.1
+      set-cookie:
+      - dhusAuth=ea1a3b69907ce71cfb5090a665dfccf1; Path=/; HttpOnly
+      - dhusIntegrity=0b74759a8fe713f5e7b28bc2ddd71975269dc3e6; Path=/; HttpOnly
+      - JSESSIONID=5F6B9F9DF5B8C796F6AEFA7BC358EA9A; Path=/; HttpOnly
```

Comment:
VCR.py changed their header naming to lower-case, breaking our scrubbing of cookies in cassettes. I fixed it in #375, but it has not been merged yet.
Please re-recorded the cassette with these fixes when they get merged. Even better if you can rebase your git history to purge this info from the repo.

Reply:
Rerecording the test and ran into a strange error with vcr itself. It seems to run into a problem while creating the yaml file, some dictionary doesn't have a matching key. I've attached a screenshot of the output. Have any of you had this problem before?
![Screenshot from 2020-06-29 10-00-40](https://user-images.githubusercontent.com/8418461/86015601-c617eb00-b9ef-11ea-941c-c31ad58da9eb.png)

Reply:
I've noticed its also the error that caused the CI to fail on the merge commit

Reply:
That's a bug, indeed. Added a fix in #379.

Reply:
done